python/cog/server/worker.py

```diff
@@ -141,6 +141,19 @@ def _wait(
             )
 
 
+class LockedConn:
```

Comment:
Any other methods we need to ensure compatibility with `multiprocessing.Lock`?

Reply:
Sorry, yeah — that's what I meant. Just thinking out loud in case there were any implicit requirements that we should cover.